test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_String.java

```diff
@@ -1860,6 +1860,18 @@ public void test_join2() {
 		}
 	}
 
+	/**
+	 * @tests java.lang.String#replaceAll(String, String)
+	 */
+	@Test
+	public void test_replaceAll() {
+		String replace1 = "1a2a3a\u0011";
+		String result1 = replace1.replaceAll("\u1161", "[");
+		AssertJUnit.assertTrue("replaceAll() compact result should be identical", replace1 == result1);
+		String replace2 = "1a2b3c\u1161";
+		String result2 = replace2.replaceAll("\u1162", "\u1234");
+		AssertJUnit.assertTrue("replaceAll() non-compact result should be identical", replace1 == result1);
```

Comment:
This should use `assertSame()` which gives better feedback on failures.
Please add a blank line after line 1874.

Reply:
updated

Reply:
I see you also fixed the copy/paste problem in the second test.